build.sbt

```diff
@@ -28,17 +28,17 @@ val versionFunc = () => {
 
 lazy val root = (project in file("."))
     .settings(
-        description          := "'Containerized exchange-api'", 
-        name                 := "amd64_exchange-api", 
-        organization         := "com.horizon", 
-        release              := "4.1.0", 
-        resolvers            += Classpaths.typesafeReleases, 
-        scalaVersion         := "2.12.10",     // tried updating to scala 2.13.1, but got many compile errors in intellij related to JavaConverters being deprecated
-        summary              := "'Open Horizon exchange-api image'", 
-        vendor               := "IBM", 
-        version              := versionFunc(), 
+        description                   := "'Containerized exchange-api'",
+        name                          := "amd64_exchange-api",
+        organization                  := "com.horizon",
+        release                       := versionFunc(),
```

Comment:
I'm probably just out of the loop on this, but we want the `release` to just be whatever version the exchange is at? Isn't that going to get confusing?

Reply:
As far as I am aware Open Horizon as a whole does not have a version, so the only other thing we have is the version of the Exchange.

Reply:
Ah ok, I understand now, thanks!